### ISSUE TITLE
LeadingZero fails inside of maps

### ISSUE THREAD

**Issue by benesch** [brigade/scss-lint#151]
Given the following SCSS:

``` scss
$sizes: (
  navbar: (0.10, 0.20, 0.30)
);

p {
  margin: 0.40;
}
```

`scss-lint` only complains about the leading zero in the `p` selector and fails to detect the leading zeros in the `$sizes` map.

Looks like it might be Sass's fault—these are the results of `source_from_range(node.source_range)` on each node given to the leading zero linter's [`visit_script_number`](https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/leading_zero.rb#L18):

```
".10, "
".20, "
".30)\n"
```

Each is curiously shifted one character to the right, chopping off the leading zero.

Thoughts? I really don't know how to go about investigating this further.
